packages/sdk-multichain/src/multichain/rpc/client.test.ts

```diff
@@ -58,7 +57,7 @@ t.describe('RPCClient', () => {
 			api: {
 				infuraAPIKey: 'test-infura-key',
 				readonlyRPCMap: {
-					'eip155:1': 'https://custom-mainnet.com',
+					'eip155:11155111': 'https://custom-mainnet.com',
```

Comment:
[nit] this wouldn't be mainnet anymore hehe

Reply:
I changed this to Sepolia instead because some of the other tests were operating against `eip155:1` which would be using a different rpc url than expected. This way I don't need to update the other tests that assume `eip155:1` is an infura endpoint